TheMovieDb/Presenter/HomeViewPresenter.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  PresenterViewTable.swift
+//  TheMovieDb
+//
+//  Created by Jonathan Hernandez on 08/11/21.
+//
+
+import UIKit
+
+protocol HomeViewPresenter {
+    func fetchAllMovieList()
+}
+
+class HomeViewPresenterImp: HomeViewPresenter {
+    
+    weak var viewHome: HomeView?
+    private let clientMoive: MovieDBClient? = MovieDBClient()
+    private var dctMoviesBySection: [SectionMovie: [Movie]] = [:]
```

Comment:
This is a good first approach to pass the data (`Movie`) to the view.

Let's try to create a `MovieViewModel` and use the adapter pattern to transform your models from `Movie` to `MovieViewModel`.